content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md

```diff
@@ -12,11 +12,6 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
-## Forking gists
-
-Each gist indicates which forks have activity, making it easy to find interesting changes from others.
-
-![Gist forks](/assets/images/help/gist/gist_forks.png)
 
 ## Cloning gists
 
```

Comment:
Any information in a screenshot should be in text as well. Let's add a sentence before this screenshot, saying something like "To clone a gist, select the **Embed** dropdown menu, then click **Clone via HTTPS** or **Clone via SSH**."

We also need to update the alt text, and fix the outline (which is too thick).